### DISCUSSION TITLE
Auto mount and unmount the drive on scheduled intervals.

### DISCUSSION THREAD

**Discussion by srinivassambari1** [iterate-ch/cyberduck#16289] [Ideas]
My application has a job to run from 08 am to 08pm, once the schedule is over for a day next day it is not able to detect the drive created by mountain duck and needs me to unmount and mount the drive manually on every day so my application wil continue to fetch files from the drive. 

I would like to know if there is any possibilities to auto unmount and mount the drive on a give scheduled time on everyday. 

I am using Mountain duck with S3 connection on a windows machine.

**Comment by tatze96** [comment]
This may be possible with the help of a script. However, disconnecting the drive without checking whether all files were transferred may lead to file loss if you are using the *Online* mode.

You can find the commands to [mount/connect](https://docs.cyberduck.io/mountainduck/interface/#connect-using-the-command-line-interface) and [unmount/disconnect](https://docs.cyberduck.io/mountainduck/interface/#disconnect) bookmarks in the corresponding sections of the *User Interface* wiki page.

**Reply by muescha** [reply]
(on macOS) with the mount part it always opens only the "connect" dialog :( but not connect direct without dialog
